app/src/main/java/com/poupa/vinylmusicplayer/model/Song.java

```diff
@@ -18,7 +18,7 @@
  * @author Karim Abou Zeid (kabouzeid)
  */
 public class Song implements Parcelable {
-    public static final Song EMPTY_SONG = new Song(-1, "", -1, -1, -1, "", -1, -1, -1, "", -1, Arrays.asList(""));
+    public static final Song EMPTY_SONG = new Song(-1, "", -1, -1, -1, "", -1, -1, -1, "", -1, new ArrayList<>(Arrays.asList("")));
```

Comment:
I dont think we need a mutable list here, but just in case I've converted a non-mutable list to a mutable one